### DISCUSSION TITLE
New user. Is there a way to map mouse back and forward buttons to specific apps?

### DISCUSSION THREAD

**Discussion by Dave-8080** [noah-nuebling/mac-mouse-fix#336] [Questions] [answered]
Have mapped my Logitech Vertical mouse to Cmd] and Cmd[ for forward and back, which works fine for Safari and MS Edge. Is there a way to map these to different keys for Preview and Books?

Thanks

**Comment by noah-nuebling** [comment, marked as answer]
Hi @Dave-8080, thanks for the feedback! 

This has been a much requested feature, and it's good that you also requested it so it gets the visibility that it deserves. 

I'm thinking about how to implement this, but it's not coming any time soon, as it's not super easy to do and would require pretty substantial refactoring of the app. 

I'll mark this as answered to clean things up. You can refer to #85 as a dedicated place for discussion around this feature.
